### ISSUE TITLE
Some problem for accuracy

### ISSUE THREAD

**Issue by zhouyuan888888** [talhanai/redbud-tree-depression#4]
Hi, sorry for bothering you. I try to reproduce the experimental results about your work, but I came across a problem. Without any change for your code, the predictions for train and validation are all negative samples:
- dev_pred:[[0.30978903],........,[0.30978903]]
- pred_train_adudio:[[0.30978903],........,[0.30978903]].

In particular, as for the training process, 1) the loss is optimized from 0.6555 to 0.5990; 2) the accuracy is fixed to 0.7143 the rate between negative samples and all samples.

Can you help me to solve this problem? Thank you so much~~^_^

**Comment by talhanai** [comment]
> the predictions for train and validation are all negative samples

Can you clarify what you mean by "negative samples"?

**Comment by zhouyuan888888** [comment]
Positive means the label is 1 (depressed) and the negative means the label is 0 (not depressed). In the binary classification task, we find all the training samples and validation samples are predicted as 0 (not depressed). Moreover, the training process has two abnormal phenomenons:
- the loss is optimized from 0.6555 to 0.5990. I think the model is not trained well.
- the accuracy is fixed to 0.7143 the rates between the depressed samples (i.e., its label is 0) and all samples.

**Comment by talhanai** [comment]
> Positive means the label is 1 (depressed) and the negative means the label is 0 (not depressed). In the binary classification task, we find all the training samples and validation samples are predicted as 0 (not depressed).

Assuming that 0.5 is the probability threshold, then yes, it seems your probabilities predict zero (from the four values you shared).

> the loss is optimized from 0.6555 to 0.5990. I think the model is not trained well.

I'm not sure I can speak to what the loss value ought to be. I don't have insight into that.

> the accuracy is fixed to 0.7143 the rates between the depressed samples (i.e., its label is 0) and all samples

If the model isn't learning anything, you might have to adjust the hyperparameters. Neural networks are very finicky. Furthermore, your features are unlikely to be _exactly_ the same as the features that was used in the work, so that reason alone means you will likely have to explore a large set of hyperparameters to find the settings in which the network will learn something.

**Comment by zhouyuan888888** [comment]
Thank you so much for your suggestions~~^_^

**Comment by wuqing778** [comment]
> 非常感谢您的建议~~ ^ _ ^

您好，是否有任何代码可以生成训练数据，或者根据自己的文章编写自己的代码
